rouskinhf/datapoint.py

```diff
@@ -305,9 +313,12 @@ def from_json_line(ref, d, predict_structure=False, predict_dms=False):
         # create the datapoint
         sequence = d["sequence"]
         sequence = standardize_sequence(sequence)
+        shape = d["shape"] if "shape" in d else None
 
         if predict_structure and (not "structure" in d) and (not "paired_bases" in d):
-            d["structure"] = RNAstructure_singleton.predictStructure(sequence, dms=d["dms"] if "dms" in d else None)
+            if 'dms' in d and 'shape' in d:
+                print('DMS and SHAPE are both present in the json line. Using DMS.')
```

Comment:
@AlbericDeLajarte we're using DMS by default if there are two data types in the dataset? or shall we just raise an error? It's an unlikely scenario anyways

Reply:
Yeah for now we can raise an error, I guess all our dataset are separated into SHAPE or DMS